src/SurveyService.Data/GroupRepository.cs

```diff
@@ -0,0 +1,29 @@
+using LT.DigitalOffice.SurveyService.Data.Interfaces;
+using LT.DigitalOffice.SurveyService.Data.Provider;
+using LT.DigitalOffice.SurveyService.Models.Db;
+using System;
+using System.Threading.Tasks;
+
+namespace LT.DigitalOffice.SurveyService.Data;
+
+public class GroupRepository : IGroupRepository
+{
+  private readonly IDataProvider _provider;
+
+  public GroupRepository(IDataProvider provider)
+  {
+    _provider = provider;
+  }
+
+  public async Task<Guid?> CreateAsync(DbGroup dbGroup)
+  {
+    if (dbGroup is null)
+    {
+      return null;
+    }
+
+    _provider.Groups.Add(dbGroup);
+    await _provider.SaveAsync();
+    return dbGroup.Id;
```

Comment:
May be add emty line? like in others methods?

Reply:
Done